wasm/src/main/scala/wasm4s/WasmContext.scala

```diff
@@ -306,6 +329,15 @@ class WasmContext(val module: WasmModule) extends FunctionTypeWriterWasmContext
       addFunction(startFunction)
       module.setStartFunction(WasmFunctionName.start)
     }
+
+    // Aggregated Elements
+
+    if (_funcDeclarations.nonEmpty) {
+      val exprs = _funcDeclarations.toList.map { name =>
+        WasmExpr(List(WasmInstr.REF_FUNC(WasmImmediate.FuncIdx(name))))
+      }
+      module.addElement(WasmElement(WasmFuncRef, exprs, WasmElement.Mode.Declarative))
```

Comment:
[question]
Why do we need to add some functions into element section?

Reply:
I added a comment. Hopefully that's clearer now.

Reply:
Thanks! Oh, I didn't know about declarative segment